### ISSUE TITLE
The codebase can't be run when using the "preview" command.

### ISSUE THREAD

**Issue by anilnatha** [NASA-PDS/portal-wp#19]
### 💡 Description

When the command `npm run preview` is used, the portal code can't be accessed in a browser due to an error.

<img width="2003" alt="image" src="https://github.com/user-attachments/assets/6a20cbf2-f316-42ab-ab7a-dfe8d18e4637">

**Comment by anilnatha** [comment]
After researching this issue and trying a number of fixes, nothing worked.

In an attempt to better understand the issue I created a test package and test app which are configured very similar to our wds-react and portal-wp codebases. Incrementally I added functionality until I was able to replicate the issue which occurred when Material UI components were integrated into the package and app, and referenced in the app. Adding the material ui components to the package didn't not cause an error, only when those components are referenced in the app and run in preview mode.

The repos for these test codebases are [test-react-package](https://github.com/NASA-PDS/test-react-package) and [test-react-app](https://github.com/NASA-PDS/test-react-app).
